### PR TITLE
Sscs 4282 add details to schema

### PR DESCRIPTION
**Pull request by iguannago**
This ticket is about adding DOB for both Appellant and Appointee.

### CODE REVIEW

File: src/main/resources/schema/sscs-robotics.json [hunk 1]
```diff
@@ -97,7 +97,9 @@
           "type": "string",
           "pattern": "^[0-9\\-+ ]{10,17}$"
         },
-        "email": { "type": "string" }
+        "email": { "type": "string" },
+        "isAppointee": {"$ref": "#/definitions/yesOrNo"},
```
Comment (jackmaloney) [inline src/main/resources/schema/sscs-robotics.json]: Is this field needed? I thought this was the one we removed from our data structure (which was put back in to handle old case data)?

File: src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java [hunk 2]
```diff
@@ -33,7 +33,7 @@ public UploadResponse upload(List<MultipartFile> files) {
 
         try {
             return documentUploadClientApi
-                    .upload(DUMMY_OAUTH_2_TOKEN, serviceAuthorization, files);
+                    .upload(OAUTH_2_TOKEN, serviceAuthorization, files);
         } catch (HttpClientErrorException httpClientErrorException) {
             throw new UnsupportedDocumentTypeException(httpClientErrorException);
         }
```
Comment (sathishbabuj) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java]: This exception should be logged.

File: src/test/resources/schema/valid_robotics_agreed.json [hunk 3]
```diff
@@ -10,7 +10,9 @@
     "phoneNumber": "07683736513",
     "postCode": "CM12 0HN",
     "title": "Mr",
-    "townOrCity": "Perferendis nemo id velit dolore qui officia"
+    "townOrCity": "Perferendis nemo id velit dolore qui officia",
+    "isAppointee": "No",
```
Comment (jackmaloney) [inline src/test/resources/schema/valid_robotics_agreed.json]: Please explain what the point of this is? It makes no sense to me

File: src/main/resources/schema/sscs-robotics.json (hunk 1 above)
Comment (iguannago) [reply src/main/resources/schema/sscs-robotics.json]: it's removed now. Thanks.

File: src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java (hunk 2 above)
Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java]: it's done now. thanks.

File: src/test/resources/schema/valid_robotics_agreed.json (hunk 3 above)
Comment (iguannago) [reply src/test/resources/schema/valid_robotics_agreed.json]: It's removed now. Thanks.

File: src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java [hunk 4]
```diff
@@ -32,11 +34,21 @@ public UploadResponse upload(List<MultipartFile> files) {
         String serviceAuthorization = authTokenGenerator.generate();
 
         try {
-            return documentUploadClientApi
-                    .upload(DUMMY_OAUTH_2_TOKEN, serviceAuthorization, files);
+            return documentUploadClientApi.upload(OAUTH_2_TOKEN, serviceAuthorization, files);
         } catch (HttpClientErrorException httpClientErrorException) {
+            log.error("Doc Store service failed to upload documents...");
```
Comment (sathishbabuj) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java]: Log the actual  exception httpClientErrorException.

Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java]: done

**Pull request merged**